core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/drs/DrsHubResolver.scala

```diff
@@ -45,16 +44,20 @@ class DrsHubResolver(drsHubUrl: String)(implicit
     }
   }
 
-  override def drsServiceAccountEmail(drsUrl: String, userInfo: UserInfo): Future[Option[String]] =
+  override def drsSignedUrl(drsUrl: String, userInfo: UserInfo): Future[Option[String]] =
```

Comment:
why does this return an option? It looks like it throws an exception rather than return `None`

Reply:
ah I think I changed behavior since defining the signature